### DISCUSSION TITLE
FREETZ_FWMOD_SIGN

### DISCUSSION THREAD

**Discussion by Sven-Moennich** [Freetz-NG/freetz-ng#532] [Unerfüllte Wünsche]
Moin.
Ich bitte darum der menuconfig unter FREETZ_FWMOD_SIGN noch Möglichkeiten für die Nutzung einer eigenen CA hinzu zu fügen.
Insbesondere vermisste ich das angeben einer Datei für das Password.
Aber auch eigene Pfad Angaben zu den Keys wäre nett.

Ich hab mir das erst mal notdürftig incomplete erweitert:
![Screenshot 2022-05-08 16 44 03](https://user-images.githubusercontent.com/36826629/167301659-7e44c5a9-a4a9-4fe6-a80a-4123c356ac55.png)

Die Dateinamen prv und pub würde ich nach meinem Geschmack mit Dateiendung versehen.

**Comment by fda77** [comment]
Ersetz die Datein prv und pub in ~/.fr..... einfach durch Links die auf deine Dateien zeigen, welche Endung diese auch immer haben

**Reply by Sven-Moennich** [reply]
Ja, hat vorteile wenn man keine zentrale konfiguration hat.
